### ISSUE TITLE
Change Target Framework of Orleans.TelemetryConsumers.NewRelic to .NET Standard

### ISSUE THREAD

**Issue by enashed** [dotnet/orleans#5044]
Orleans.TelemetryConsumers.NewRelic currently targets .NET Framework 4.6.1

Can we please change that project to target .NET Standard ?

Cheers

**Comment by ReubenBond** [comment]
Good idea. We will need to bump up the package version (defined in `Directory.Build.props`), too.

Would you mind submitting a PR?
